mantis-common/src/main/java/io/mantisrx/common/MantisProperties.java

```diff
@@ -28,24 +28,40 @@ private MantisProperties() {
         env = System.getenv();
     }
 
+    @Deprecated
```

Comment:
@Andyz26 Please take a look. I fixed MantisProperties to properly fallback to System.getProperty and then System.getEnv. I believe you intend for this class to be used with static methods going forward. So I'm deprecating all the instance methods.

Reply:
You need System properties and env var because that's how some jobs get job parameters.